### PR TITLE
add testFilterZtoA

### PR DESCRIPTION
**Pull request by AlyonkaNY**

### CODE REVIEW

**Review by IrynaKolh — APPROVED** [review]
LGTM

**Review by ArinaJur — APPROVED** [review]

**Review by Lokankara — APPROVED** [review]

**Review by ystaros — APPROVED** [review]
LGTM

**Review by Volhabo — APPROVED** [review]
LGTM

File: src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java [hunk 1]
```diff
@@ -11,11 +11,8 @@
 import school.redrover.runner.BaseTest;
 
 import java.time.Duration;
-import java.util.HashMap;
-import java.util.ArrayList;
+import java.util.*;
```
Comment (Lokankara) [inline src/test/java/school/redrover/GroupJavaExitCodeZeroTest.java:14]: config  wildcard imports
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

**Pull request merged**